TempleEight/docs/arch/system.md

```diff
@@ -3,3 +3,190 @@ id: system
 title: System Architecture
 sidebar_label: System Architecture
 ---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+On top of generating service-level application code, Temple produces orchestration configuration to enable executing the services together as a whole system.
+This guide will take you through how a Temple generated application is designed on the super-service level.
+We'll look at how the system is organised and how it changes with more advanced features.
+
+The details of how the application code in each service is architected can be seen in the [Service Architecture](service) reference.
+
+## Minimal Example
+
+We'll start with the example Templefile from the [Getting Started](../getting-started) Guide.
+
+```templefile
+ExampleProject: project {
+  #language(go);
+  #database(postgres);
+  #provider(dockerCompose);
+}
+
+ExampleService: service {
+  foo: string;
+  bar: int;
+}
+```
+
+Also from the Getting Started Guide is this diagram outlining the system architecture:
+
+<img alt="Tutorial System Architecture" src={useBaseUrl('img/tutorial-architecture.png')} />
+
+It shows a high level description of the system, detailing how the [Kong API Gateway](https://konghq.com/kong/) sits between the user and the backend services, and how the service communicates with it's database.
+
+Let's delve a little bit deeper into this architecture and see some finer details:
+
+<img alt="System Architecture Deeper" src={useBaseUrl('img/system-architecture-basic.png')} />
+
+We can see that all components of our application are executing inside of [Docker](https://www.docker.com/) containers.
+This includes Kong, and allows us to not have to worry about it's installation, since our orchestration tool deals with this for us. 
+Containers allow us to encapsulate our application components and their dependencies, allowing them to be easily ran anywhere, among many other benefits.
+
+We can also see that Kong uses it's own database to store information about which services exist and how to route requests to them.
+However, this is managed by the orchestration platform automatically, so we don't need to worry about it.
```

Comment:
Saying "we don't need to worry about it" in a guide about system architecture feels a little off? The whole point is that we deal with it?